src/App/Action/UserTodoList.php

```diff
@@ -49,6 +49,7 @@ public function __construct(TemplateRendererInterface $templates, QueryBus $quer
      * @param ServerRequestInterface $request
      * @param ResponseInterface $response
      * @param callable $next
+     * @return HtmlResponse
```

Comment:
I would prefer to match any possible interface that could implementend in the future for middlewares. So it should be ResponseInterface instead of HtmlResponse.